lib/examples/compare-chromedp/scroll_screenshot/main.go

```diff
@@ -0,0 +1,29 @@
+// Package main ...
+package main
```

Comment:
Could you move this to this file: https://github.com/go-rod/rod/blob/main/examples_test.go

Reply:
I'm finished. Any other questions?